internal/regexp/runes.go

```diff
@@ -0,0 +1,211 @@
+// Copyright 2019 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package regexp
+
+import (
+	"regexp/syntax"
+	"strings"
+)
+
+// CompileAcme is like Compile but treats ^ and $ as only matching
```

Comment:
I'm hoping (assuming even?) that your change fixes the annoyance that something like `Edit .,.+5 x/^/ i://:` currently adds a `//` on one extra line?

Reply:
I think that edit line always worked correctly (added 5 `//`). However, it now correctly handles something like this: `Edit .,.+5 x/^	/ i://:` (with a tab after the `^`). If your dot was in the middle of a line, immediately before a tab, it was indeed adding an extra `//`.